src/commands/createNewProject/ProjectCreateStep/ScriptProjectCreateStep.ts

```diff
@@ -82,6 +88,8 @@ __azurite_db*__.json`));
         };
 
         await bundleFeedUtils.addDefaultBundle(context, hostJson);
+        if (context.languageModel === nodejsNewModelVersion)
```

Comment:
Can we use braces here? Non-block-style `if`s too often lead to hard-to-spot bugs...